integration-tests/utils/integration-utils.js

```diff
@@ -365,7 +365,8 @@ const logPageAjaxResponses = page => {
   const terraRequests = [
     'broad',
     'terra',
-    'googleapis'
+    'googleapis',
+    'bvdp'
```

Comment:
Missed it before, but we should also capture `bvdp-*` UI server requests and write them to test log.

Reply:
What requests are made to `bvdp-*`? Is it the `config.json` file?